packages/web/sentry.client.config.ts

```diff
@@ -4,14 +4,30 @@
 
 import * as Sentry from "@sentry/nextjs";
 
+import { getValidSwapTRPCRoutesForSentry } from "~/utils/sentry-init";
+
 Sentry.init({
   dsn: "https://c696452bb7ce4cc98150142ebea1c32f@o4505285755600896.ingest.us.sentry.io/4505285757698048",
 
   environment:
     process.env.NEXT_PUBLIC_VERCEL_ENV || process.env.NODE_ENV || "development",
 
   // Adjust this value in production, or use tracesSampler for greater control
-  tracesSampleRate: 0.05,
+  tracesSampler: (samplingContext) => {
+    const validPaths = ["/", "/pool/[id]", "/assets/[denom]"];
+    const validTrpcRoutes = getValidSwapTRPCRoutesForSentry();
```

Comment:
Why are tRPC routes relevant in the client?

Reply:
In our FE, tRPC routers can run locally. For example, we have the [localRouter](https://github.com/osmosis-labs/osmosis-frontend/blob/1edcc9b92711303a65d3cb69415d9604b1d4832d/packages/web/server/api/root-router.ts#L13-L16) that includes all of these functions. Since they are local, they begin their trace in the browser.